README.md

```diff
@@ -160,3 +160,36 @@ HTML example:
 ``` html
 <button data-xpaystation-widget-open>Buy Credits</button>
 ```
+
+
+
+RUSSIAN
+
+ENGLISH
+
+SPANISH
+
+ENGLISH
+
+RUSSIAN
+
+SPANISH
+Source text
+### Troubleshooting
+
+Browsers are sensitive to anything that could affect user security.
+You should carefully work with [user trusted events](https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted), e.g., clicks handling.
+If payment method opens in the new browser tab, you must keep relation between trusted event and new tab opening functionality.
+If the conditions for opening a new tab depends on asynchronous code, it will break user event trusting will result in an error.
+Refactoring the callback code that opens a new tab in synchronous style will solve this problem.
```

Comment:
Refactoring the callback code that opens a new tab in synchronous style can resolve this problem.